src/frontmatter.xml

```diff
@@ -35,84 +35,87 @@
 
     <colophon>
 
-        <copyright>
-            <year>2010<ndash />2017</year>
-            <holder>Jack Green, Nick Chura</holder>
-            <!--<minilicense>GFDL License</minilicense>
-            <shortlicense>Permission is granted to copy, distribute and/or modify this document under the terms of the GNU Free Documentation License, Version 1.2 or any later version published by the Free Software Foundation; with no Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts.  A copy of the license is included in the appendix entitled <q>GNU Free Documentation License.</q>  All trademarks<trademark /> are the registered<registered /> marks of their respective owners.</shortlicense>-->
-        </copyright>
+      <copyright>
+          <year>2012<ndash />2017</year>
+          <holder>Jack Green, Nickolas Chura</holder>
+          <minilicense>Creative Commons BY-SA</minilicense>
+          <shortlicense>This work is licensed under a Creative Commons Attribution-ShareAlike 4.0 International License.</shortlicense>
+          <creativecommons BY="yes" NC="yes" ubiquity="medium"/>
```

Comment:
The "creativecommons" element is not yet a thing, and the XML markup for it is not yet established. It's a side project I'm working on. (Well, I haven't worked on it for like 6 months now.) So I'd say delete this line.